docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -32,6 +32,8 @@ S3 configuration properties
         S3-compatible storage system instead of AWS. When using v4 signatures,
         it is recommended to set this to the AWS region-specific endpoint
         (e.g., ``http[s]://s3.<AWS-region>.amazonaws.com``).
+    * - ``hive.s3.region``
+      - The optional region to force the S3 client to connect to.
```

Comment:
How about

Optional property to force the S3 client to connector to the specified region only.